content/javascript/concepts/sets/sets.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: 'Set'
+Description: 'Set is a special object without keys which can only store unique values.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Set'
+  - 'Object'
+  - 'Collections'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+`Set` is an object in JavaScript that is a collection of unique values.
+
+A set object can be created with the `Set()` constructor function.
+
+```js
+const set = new Set();
+```
+
+## Main methods
+
+- `set.add(value)` - adds a value to the set. If the value is already in the set, it doesn't do anything.
+- `set.delete(value)` - removes the given value, returns `true` if the value existed in the set, otherwise `false`.
+- `set.has(value)` - returns `true` if the value exists in the set, otherwise `false`.
+- `set.clear()` - deletes everything from the set.
+- `set.size` - returns the number of elements in the set.
```

Comment:
```suggestion
- `set.add(value)`: Adds a value to the set. If the value is already in the set, it doesn't do anything.
- `set.delete(value)`: Removes the given value, returns `true` if the value existed in the set, otherwise `false`.
- `set.has(value)`: Returns `true` if the value exists in the set, otherwise `false`.
- `set.clear()`: Deletes everything from the set.
- `set.size`: Returns the number of elements in the set.
```

Just a style preference 📝